.github/workflows/gpu_signed_integer_h100_tests.yml

```diff
@@ -100,14 +100,15 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt update
-          sudo apt install -y checkinstall zlib1g-dev libssl-dev 
+          sudo apt install -y checkinstall zlib1g-dev libssl-dev libclang-dev
```

Comment:
not doing this to be annoying but to not forget the locations if we update the AMI